code/modules/scripting/Implementations/Telecomms.dm

```diff
@@ -24,14 +24,22 @@
 		interpreter.GC()
 
 
+//temp
+/datum/TCS_Compiler
+	var/datum/n_scriptOptions/nS_Options/options		
+	var/datum/n_Scanner/nS_Scanner/scanner				
+	var/list/tokens										
+	var/datum/n_Parser/nS_Parser/parser					
+	var/datum/node/BlockDefinition/GlobalBlock/program	
+
 	/* -- Compile a raw block of text -- */
 
-/datum/TCS_Compiler/proc/Compile(code as message)
-	var/datum/n_scriptOptions/nS_Options/options		= new()
-	var/datum/n_Scanner/nS_Scanner/scanner				= new(code, options)
-	var/list/tokens										= scanner.Scan()
-	var/datum/n_Parser/nS_Parser/parser					= new(tokens, options)
-	var/datum/node/BlockDefinition/GlobalBlock/program	= parser.Parse()
+/datum/TCS_Compiler/proc/Compile(list/code)
+	options		= new()

```

Comment:
This is technically unnecessary but it's also harmless (as far as I can tell) and allows more ability for debugging.